egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -299,6 +313,8 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
   extra_opts+=(--right-context $right_context)
   extra_opts+=(--valid-left-context $((chunk_width + left_context)))
   extra_opts+=(--valid-right-context $((chunk_width + right_context)))
+  extra_opts+=(--left-shift-window $left_shift_window)
+  extra_opts+=(--ivector-interval $ivector_interval)
```

Comment:
Maybe you should write ivector-interval into $egs_dir/info/ and check if ivector-interval matches the one in egs_dir in case egs-dir is provided by user.